form/src/main/java/org/dhis2/form/ui/provider/inputfield/FieldProvider.kt

```diff
@@ -623,6 +645,132 @@ private fun ProvideEmail(
     )
 }
 
+@Composable
+private fun ProvideMatrixInput(
```

Comment:
Please, could you extract this provides to an external class to reduce FieldProvider size?.

Reply:
Of course, I'll update it